source/components/_footer.erb

```diff
@@ -0,0 +1,29 @@
+<footer class="app-c-footer">
+  <div class="app-c-footer__navigation govuk-o-grid">
+    <div class="govuk-o-grid__item govuk-u-mb-5 govuk-o-grid__item--one-quarter">
+      <h2 class="app-c-footer__heading heading-medium">GOV.UK Design System</h2>
+      <ul class="app-c-footer__list govuk-c-list">
+        <li><a href="/design-patterns/patterns/sitemap.html">Site contents</a></li>
```

Comment:
Should we omit these until the pages exist?

Reply:
The footer links? Sure, we can remove them. They were good to test styles with.